app/models/concerns/hyrax/file_set/belongs_to_works.rb

```diff
@@ -7,8 +7,10 @@ module BelongsToWorks
       #   before_destroy :remove_representative_relationship
       # end
 
+      # Objects that call this FileSet a member
       def parents
-        in_works
+        query_service = Valkyrie::MetadataAdapter.find(:indexing_persister).query_service
+        query_service.find_inverse_references_by(resource: self, property: :member_ids)
```

Comment:
Could this be simplified to `query_service.parents(resource: self)`?  If so, do we even need this method/module at all?

Reply:
I've made the suggested change and deprecated this method.